app/components/Post.tsx

```
@@ -69,6 +145,44 @@ export default function Post({
+                  initial={{ height: 0, opacity: 0 }}
+                  animate={{ height: "auto", opacity: 1 }}
+                  exit={{ height: 0, opacity: 0 }}
+                  transition={{ duration: 0.1 }}
+                >
+                  <li className={styles.postMenuItem}>
+                    <button
+                      onClick={() => {
+                        setIsMenuOpen(false);
+                        handleDeletePost();
+                      }}
+                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)